react-client/src/components/Common/Login/LoginBtn/LoginBtn.js

```diff
@@ -0,0 +1,31 @@
+import style from './LoginBtn.module.scss'
+import React from 'react'
+import Proptypes from 'prop-types'
+import { useNavigate } from 'react-router'
+
+const LoginBtn = ({ text }) => {
+  const navigate = useNavigate()
+
+  const btnClicked = () => {
+    if (text === 'Logout') {
```

Comment:
What if someone passes "logout" to this component and not "Logout"? or even "log out"? it's a problem to set the functionality based on the text. not a good practice. I would pass a text and a handler function for each option - login or logout